wallet/src/Pos/Wallet/Aeson/ClientTypes.hs

```diff
@@ -38,7 +37,6 @@ deriveJSON defaultOptions ''CWalletInit
 deriveJSON defaultOptions ''CPaperVendWalletRedeem
 deriveJSON defaultOptions ''CTxMeta
 deriveJSON defaultOptions ''CProfile
-deriveJSON defaultOptions ''BackupPhrase
```

Comment:
I believe this would be a breaking change for V0 as json serilization will change to your manually written instances of ToJson/FromJson .

Reply:
It _shouldn't_ since the manually written one preserve the same form. Only difference is that is will now fail to parse invalid mnemonic, getting an early error rather than holding a potentially wrong list of words that only explodes when we start using it. 
For any valid inputs however, the structure of the `ToJSON` and `FromJSON` instances have been preserved (a list of strings).

Reply:
To provide evidence of safety, would you be willing to write a few golden tests?